.travis.yml

```
@@ -0,0 +1,48 @@
+language: java
+
+cache:
+  directories:
+    $HOME/.m2
+
+env:
+  - JAVA_TOOL_OPTIONS=-Dhttps.protocols=TLSv1.2
+
+git:
+  depth: 1
```

Although having a git depth of 1 saves a tiny amount of time, I think @davidgrayston and I discussed that we would rather have an update depth of 3, so that we can better track our failing metrics (i.e. we won't get as many `failed` builds because of another commit superseding it). We still have a depth of 1 in some travis files, but the plan is to change to 3 I think.
```suggestion
  depth: 3
```